tests/test_conversations.py

```diff
@@ -0,0 +1,251 @@
+"""
+This module contains tests related to helpdesk, finaid, etc.
+"""
+
+from django.core.urlresolvers import reverse
+from django.conf import settings
+
+from conference.models import Conference
+from conversations.models import Thread
+from conversations.common_actions import ThreadActions
+from conversations.user_interface import UserNewFinaidRequest
+from tests.common_tools import make_user, redirects_to, template_used
+
+
+def _setup(client):
+    Conference.objects.create(code=settings.CONFERENCE_CONFERENCE)
+    make_user(email="joe@example.com", password="foobar")
+    client.login(email="joe@example.com", password="foobar")
+
+
+def _setup_admin(client):
+    Conference.objects.create(code=settings.CONFERENCE_CONFERENCE)
+    make_user(email="joe@example.com", password="foobar", is_staff=True)
+    client.login(email="joe@example.com", password="foobar")
+
+
+def test_user_panel_contains_a_link_to_user_threads(db, client):
```

Comment:
Is there a difference between marking a test with `django_db` and passing a db fixture?

Reply:
i think i accidentally answered above. not sure if there is (in this case) and the fixture seems cleaner. :)